<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerBG</class>
 <widget class="QWidget" name="PlayerBG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPlayerBG</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CPlayerWidget" name="PlayerWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelCurTime">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CPlayerSlider" name="sliderPlayer">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDuration">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmboxSpeed"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack5s">
         <property name="toolTip">
          <string>ctrl+左方向键</string>
         </property>
         <property name="text">
          <string>快退5s</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="toolTip">
          <string>左方向键</string>
         </property>
         <property name="text">
          <string>上一帧</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnPlay">
         <property name="toolTip">
          <string>空格键</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForward">
         <property name="toolTip">
          <string>右方向键</string>
         </property>
         <property name="text">
          <string>下一帧</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForward5s">
         <property name="toolTip">
          <string>ctrl+右方向键</string>
         </property>
         <property name="text">
          <string>快进5s</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTaken">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>开始提取</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CPlayerSlider</class>
   <extends>QSlider</extends>
   <header>PlayerSlider.h</header>
  </customwidget>
  <customwidget>
   <class>CPlayerWidget</class>
   <extends>QWidget</extends>
   <header>Playerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
